src/modules/guide.js

```diff
@@ -47,5 +54,6 @@ function onKeyUp( event ) {
 }
 
 export default function addGuide( countdown = 2500 ) {
+	countdown = getOptions().steps[0].startDelay || countdown;
```

Comment:
Is it guaranteed that the `steps` option array always exist and have at least one item? Otherwise, some test code should come first.

Reply:
Yes, if showGuide exists, there must be steps in the same array. I think there's no need to add extra ifs here.